packages/framework/esm-styleguide/src/_vars.scss

```
@@ -27,6 +27,12 @@ $field-01: #f4f4f4;
 $grey-2: #e0e0e0;
 $labeldropdown: #c6c6c6;
 
+:root {
+  --brand-01: #005d5d;
+  --brand-02: #004144;
+  --brand-teal-01: #007d79;
+}
```

Since this is in an scss file, wouldn't it make sense to do:

```suggestion
:root {
  --brand-01: $brand-01;
  --brand-02: $brand-02;
  --brand-teal-01: $brand-teal-01;
}
```

---

Secondary point to all of this, but `brand-teal-01` could use a rename...

---

Strangely, that doesn't work—if you do this then the variable `--brand-01` gets set to the string literal `"$brand-01"`.

---

Hmmm... I think `brand-light-01` at least gets the goal across better? I'm thinking that we'll definitely need some sort of documentation with annotated screenshots so people can see where the branding takes place.